### PR TITLE
blub blub, here comes a core plugin

### PR DESCRIPTION
**Pull request by condret**
- [ ] Mark this if you consider it ready to merge
- [ ] I've added tests (optional)
- [ ] I wrote some lines in the [book](https://github.com/radareorg/radare2book) (optional)

**Description**

**Copilot**

### CODE REVIEW

File: libr/core/p/core_agD.c [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+#else
+	.name = "agD",
+	.desc = "agD core plugin",
+	.license = "LGPL3",
+	.author = "condret",
+#endif
+	.call = r_cmd_agD_call,
+};
+
+#ifndef CORELIB
```
Comment (trufae) [inline libr/core/p/core_agD.c:76]:
```suggestion
#ifndef R2_CORELIB
```

File: libr/core/p/core_agD.c [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+	r_agraph_print (dtagraph);
+	r_agraph_free (dtagraph);
+	r_graph_free (fcn_dtgraph);
+	r_graph_free (fcn_graph);
+	r_cons_flush ();
+	return true;
+}
+
+RCorePlugin r_core_plugin_agD = {
+#if R2_VERSION_NUMBER > 50808
```
Comment (trufae) [inline libr/core/p/core_agD.c:60]: Why not just in cmd_anal or an anal plug instead of core?

Comment (condret) [reply libr/core/p/core_agD.c:60]: because this is only doing this one specific command

Comment (condret) [reply libr/core/p/core_agD.c:60]: well, i don't consider this something actually useful, outside of debugging the domtree code. nobody needs to see a domtree. this is only useful for improving analysis

**Pull request merged**